src/test/components/ActiveTabTimeline.test.js

```diff
@@ -148,5 +152,80 @@ describe('ActiveTabTimeline', function() {
       fireEvent.click(getGlobalTrackRow());
       expect(getSelectedThreadIndex(getState())).toBe(threadIndex);
     });
+
+    it('does not display the resources panel if there are no resource tracks', () => {
+      const { getState, container } = setup();
+      expect(getActiveTabResourceTracks(getState()).length).toBe(0);
+      expect(container.querySelector('.timelineResourcesHeader')).toBeFalsy();
```

Comment:
I think this would be better to select the container by text, rather than class name. I believe the consensus on selecting elements is to never uses classes, and if needed use a `data-testid` attribute ([documentation](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid)). However, the best thing is to select things by user-like actions, such as the `getByText`.

This really tests user-visible behavior, and not implementation details.